source/13-transitioning/01-transitioning-to-a-new-reporting-service.rst

```diff
@@ -5,8 +5,24 @@
 Transitioning to a New Reporting Service
 ----------------------------------------
 
-When a content provider implements a new reporting service, underlying logging system, or approach, they:
+When a content provider implements a new reporting service, underlying logging system, or approach, they MUST continue to meet the requirement to offer valid COUNTER reports for the current year plus the prior 24 months (or from the date they first became compliant, whichever is later) via a web interface and via a SUSHI server.
 
-* MUST continue to meet the requirement to offer valid COUNTER reports for the current year plus the prior 24 months (or from the date they first became compliant, whichever is later) via a web interface and via a SUSHI server.
-* MUST support COUNTER reports that may include a range of months that span the transition period. If the new reporting service was deployed in August of 2024, a customer could request a report for January-December 2024 and receive a single report.
-* When it is not practical to support a single report with date ranges that span the transition period, the content provider MUST perform the transition on the first day of a month. If the new reporting service was deployed in August 2024, a customer wanting January-December 2024 usage would request January-July 2024 from the previous reporting service and August-December 2024 from the new reporting service. For clarity, a provider MUST NOT perform the transition mid-month such that the customer is required to run reports on both the old and new reporting services for the same month and merge and sum the results to obtain actual monthly usage.
+
+Availability of Previous Reports
+""""""""""""""""""""""""""""""""
+
+* Report providers SHOULD offer a single report with date ranges that span the transition period. That is, if the new reporting service was deployed in August of 2024 a customer could request a report for January-December 2024 and receive a single report.
+* When it is not practical to support a single report with date ranges that span the transition period, the content provider SHOULD perform the transition on the first day of a month. If the new reporting service was deployed in August 2024, a customer wanting January-December 2024 usage would request January-July 2024 from the previous reporting service and August-December 2024 from the new reporting service. 
```

Comment:
I would suggest to simply use paragraphs instead of bullet points.

Reply:
On the list for the copy-edit issue